docs/auth/ios-auth-environments.md

```
@@ -0,0 +1,68 @@
+   - Clerk test/dev publishable key.
+   - Clerk dev frontend API.
+   - Supabase dev URL and anon key.
+   - `SUPABASE_EXPECTED_HOST` matching the Supabase dev URL host.
+   - Local or dev `API_BASE_URL` and matching `API_EXPECTED_HOST`.
+3. Create `Config.xcconfig` with:
+
+```xcconfig
+#include "Config-Development.xcconfig"
+```
+
+4. Keep `ALLOW_PRODUCTION_SERVICES_IN_DEBUG = NO` unless a human explicitly approves a temporary production-provider test.
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 4; Style: 1/1/1

(MD029, ol-prefix)